### ISSUE TITLE
First Run: No Gravatar

### ISSUE THREAD

**Issue by jkmassel** [wordpress-mobile/WordPress-Android#21415]
The "Me" tab doesn't have my gravatar as an icon, it has a generic "Me" icon (that's the wrong color in dark mode)

This only happens on the first run after login – it seems to fix itself on subsequent runs

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by Akshaykomar890** [comment]
@jkmassel I started working on this issue but faced some problems. I have raised a PR and added details about the issue I’m facing. Could you help me solve this?
Here’s my PR with the changes:#21672

Would appreciate any help in debugging this. Thanks

**Comment by crazytonyli** [comment]
I can't reproduce this issue in the trunk branch. I tried on a completely new app. The avatar image would flash between the placeholder one and the real one a couple of times, but eventually, the real avatar is displayed.

**Comment by crazytonyli** [comment]
Also not reproducible on App Store v25.7.1. Closing this issue.

**Comment by crazytonyli** [comment]
Sorry, I thought this is an issue in the iOS apps 🤦‍♂

**Comment by Anubhav8176** [comment]
Is this issue is still open?

**Comment by adalpari** [comment]
> Is this issue is still open?

Yes, this issue is still open in case you would like to collaborate. Thank you!

**Comment by AnandJeyapal** [comment]
@adalpari I cannot login to Wordpress.com with email, cannot see the stats (jetpack activation is failing), cannot subscribe/report a blog.

I created the Wordpress account using jurasic ninja. Is there any additional setup I need to do for the app? Please guide.

**Comment by adalpari** [comment]
> I created the Wordpress account using jurasic ninja. Is there any additional setup I need to do for the app? Please guide.

If you want to use a Jurassic Ninja site, you should log in using the "Enter your existing site address" form.

<img width="516" height="1152" alt="Image" src="https://github.com/user-attachments/assets/7013366b-c640-4747-b018-a93a0faa2319" />

**Comment by N-Yashwitha** [comment]
Is this issue still open?

**Comment by adalpari** [comment]
> Is this issue still open?

Yes, it is.

**Comment by thirumani-vihaan** [comment]
Hey @adalpari! I know it's been a few months since the last activity here, but since there are still no active PRs, I'd love to finally knock this one out.

It sounds like the "Me" tab isn't actively observing the profile/Gravatar update right after the initial login flow completes. My plan is to dig into the login success callback and ensure we trigger a UI state refresh for the avatar so it appears on that first run without needing an app restart.

Mind if I claim this and get started on a PR?

**Comment by adalpari** [comment]
@thirumani-vihaan Please, go ahead! Thanks!

**Comment by thirumani-vihaan** [comment]
Thanks @adalpari! I've just opened PR #23103 with the fix for this. Let me know if you need anything else from my end during the review process!
